pkg/restic/common_test.go

```diff
@@ -375,3 +377,42 @@ func TestTempCredentialsFile(t *testing.T) {
 
 	assert.Equal(t, "passw0rd", string(contents))
 }
+
+func TestTempCACertFile(t *testing.T) {
+	var (
+		bslInformer = cache.NewSharedIndexInformer(nil, new(velerov1api.BackupStorageLocation), 0, cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc})
+		bslLister   = velerov1listers.NewBackupStorageLocationLister(bslInformer.GetIndexer())
+		fs          = velerotest.NewFakeFileSystem()
+		bsl         = &velerov1api.BackupStorageLocation{
+			TypeMeta: metav1.TypeMeta{},
+			ObjectMeta: metav1.ObjectMeta{
+				Namespace: "velero",
+				Name:      "default",
+			},
+			Spec: velerov1api.BackupStorageLocationSpec{
+				StorageType: velerov1api.StorageType{
+					ObjectStorage: &velerov1api.ObjectStorageLocation{CACert: []byte("cacert")},
+				},
+			},
+		}
+	)
+
+	// bsl not in lister: expect an error
+	caCert, err := GetCACert(bslLister, "velero", "default")
+	assert.Error(t, err)
+
+	// now add bsl to lister
+	require.NoError(t, bslInformer.GetStore().Add(bsl))
+
+	// bsl in lister: expect temp file to be created with cacert value
+	caCert, err = GetCACert(bslLister, "velero", "default")
+	require.NoError(t, err)
+
+	fileName, err := TempCACertFile(caCert, "default", fs)
```

Comment:
missing a `os.Remove(fileName)` to avoid creating temp files on dev machines running this test?